ocaml/typing/layouts.mli

```diff
@@ -68,6 +68,13 @@ module Sort : sig
       it is set to [value] first. *)
   val get_default_value : t -> const
 
+  type change
```

Comment:
Probably these should each get a comment (or group them together without newlines and put a comment above)